doc/source/point-in-time-recovery.rst

```diff
@@ -76,16 +82,20 @@ Re-enable |PITR| to resume saving oplog slices:
 
    $ pbm config --set pitr.enabled=true
 
+As of version 1.6.0, you can recover your database to the specific point in time using any backup snapshot, and not only the most recent one. Run the ``pbm restore`` command with the ``--base-snapshot=<backup_name>`` flag where you specify the desired backup snapshot. 
+
+To restore from any backup snapshot, |PBM| requires continuous oplog. After the backup snapshot is made and |PITR| is re-enabled, it copies the oplog saved with the backup snapshot and creates oplog slices from the end time of the latest slice to the new starting point thus making the oplog continuous.
+
 
 .. rubric:: Delete a backup
 
-When you :ref:`delete a backup <pbm.backup.delete>`, all oplog slices that relate to this backup will be deleted too. For example, you delete a backup snapshot 2020-07-24T18:13:09 while there is another snapshot
-2020-08-05T04:27:55 created after it.  |pbm-agent| deletes only oplog slices that relate to 2020-07-24T18:13:09.
+When you :ref:`delete a backup <pbm.backup.delete>`, all oplog slices that relate to this backup will be deleted too. For example, you delete a backup snapshot 2020-07-24T18:13:09 while there is another snapshot 2020-08-05T04:27:55 created after it.  |pbm-agent| deletes only oplog slices that relate to 2020-07-24T18:13:09.
```

Comment:
This part is modified entirely within PBM-652, is this change still necessary?

Reply:
I've reflected these changes in https://github.com/percona/percona-backup-mongodb/pull/568 so after the merge the doc will be consistent

Reply:
Still, I don't think we should have this paragraph anymore. As of v1.6 `pbm delete-backup` doesn't delete any pitr-chunks in any case.